vscode-dotnet-runtime-library/src/Acquisition/GlobalInstallerResolver.ts

```diff
@@ -249,6 +250,10 @@ Please report this issue so it can be remedied or investigated.`));
                             this.eventStream.post(releaseJsonErr);
                             throw releaseJsonErr.error;
                         }
+                        else
+                        {
+                            this.eventStream.post(new DotnetFileIntegrityCheckEvent(`This installer file is hosted on an expected domain https://download.visualstudio.microsoft.com/.`));
```

Comment:
https://download.visualstudio.microsoft.com/ Does it redirect to an empty page?

Reply:
That's a good thing to point out. This is in the log so it should be fine. I am a bit surprised its empty, but it is just a download server, so kinda makes sense the root is 404